tests/linearizability/model.go

```diff
@@ -96,21 +95,25 @@ func step(state EtcdState, request etcdRequest, response etcdResponse) (bool, Et
 	if state.Key != request.key {
 		panic("Multiple keys not supported")
 	}
+	if response.revision != 0 && response.revision < state.LastRevision {
+		return false, state
+	}
```

Comment:
Does it deserve a unit test to explain and validate the expected behavior?

Reply:
I have updated the unit tests.